classes/Currency.php

```diff
@@ -782,7 +782,9 @@ public function refreshLocalizedCurrencyData(array $languages, LocaleRepository
                 continue;
             }
 
+            // CLDR locale give us the CLDR reference specification
```

Comment:
I was actually thinking about something more technical and straight forward like:
```php
/** @var \PrestaShop\PrestaShop\Core\Localization\CLDR\Locale $cldrLocale */
```

Reply:
hum it is already correctly typehinted by my IDE

Reply:
think about people using emacs 😇
or who are simply reading the code on github